app/src/main/java/org/simple/clinic/textInputdatepicker/TextInputDatePickerEvent.kt

```diff
@@ -13,3 +14,7 @@ data class DayChanged(val day: String) : TextInputDatePickerEvent()
 data class MonthChanged(val month: String): TextInputDatePickerEvent()
 
 data class YearChanged(val year: String): TextInputDatePickerEvent()
+
+data class DoneClicked(val minDate: LocalDate, val maxDate: LocalDate) : TextInputDatePickerEvent() {
```

Comment:
Instead of trying to pass the min and max dates like this using an event, do you think we can pass these as constructor arguments for the validator instead? Since these don't change and are passed by screen key I think that should make it easy to validate dates or we can even add them to the model when creating to avoid passing these dates like this. WDYT?

Reply:
I think adding them to the model constructor makes more sense.